.github/workflows/publish.yaml

```diff
@@ -6,6 +6,16 @@ on:
     tags: ['*']
```

Comment:
You've got a wrong idea: `(#51)` in master commit history is generated with GitHub on merge (seems like squash commit strategy), you shouldn't do it manually

Reply:
Done, deleted `(#51)` from commit msg